app/helpers/chat_channels_helper.rb

```diff
@@ -0,0 +1,46 @@
+module ChatChannelsHelper
+  def render_unopened_json_response
```

Comment:
None of these methods technically render anything, they return records from the DB.